public/locales/es/common.json

```
@@ -25,16 +26,30 @@
     "browseScholarship": "Buscar becas",
     "about": "Acerca de",
     "contact": "Contacto",
-    "reportIssue": "Reportar un problema",
     "suggestIdea": "Sugerir una idea",
     "subscribeForUpdates": "Suscríbete para actualizaciones",
     "reachOut": "Contacta con nosotros"
   },
 
   "actions": {
-    "signIn": "Iniciar Sesión"
+    "signIn": "Iniciar Sesión",
+    "apply": "Aplicar",
+    "share": "Compartir",
+    "back": "Atras",
+    "next": "Siguiente",
+    "submit": "Submitir",
+    "copyLink": "Copiar enlace",
+    "reportIssue": "Reportar un problem"
   },
 
   "dashboard": "Tablero de control",
-  "welcome": "Bienvenido"
+  "welcome": "Bienvenido",
+  "currentlySharing": "Actualmente compartiendo",
+  "general": "General",
+  "eligibilityReqs": "Requisitos de Elegibilidad",
```

Spanish doesn't generally capitalize words in titles the way English does :P
```suggestion
  "eligibilityReqs": "Requisitos de elegibilidad",
```